handbook/engineering/code-intelligence/goals.md

```diff
@@ -1,39 +1,50 @@
-# Code Intelligence goals
+# Code Intelligence goals and priorities
 
-Progress on adoption and usage is tracked in our [Looker dashboard](https://sourcegraph.looker.com/dashboards/131).
+## Goals
 
 ### Grow LSIF adoption and usage reaching [*N<sub>0</sub>*][N0] precise code intel operations a month
 
+Progress on adoption and usage is tracked in our [Looker dashboard](https://sourcegraph.looker.com/dashboards/131).
+
 **Problem:** By default, users get basic code intel results which are often incorrect for tokens with common names (such as Get). This is solved by enabling precise code intel, which gives users faster, 100% accurate results for a definition or reference for a symbol they hover over. 
 
 **Strategy:** It is imperative that we raise adoption *deeply* within a particular language and ecosystem. We want to provide fast and precise answers about a user's _universe of code_. This can only happen when that entire universe is indexed. We intend to provide an index for _all_ such universes of code by making progress on two fronts. First, we need to raise excitement for the opportunities adopting precise code intelligence makes possible. Once a user sees what is possible, they should have little reason not to put in the effort of adoption. Second, we need to reduce the friction felt during indexing (especially during adoption). Users tend to perceive only the work we ask them to do to set up precise code intelligence and do not perceive the full benefit of precise code intelligence. Reducing the friction of indexing is a critical step in growing support. It is also necessary to raise adoption *widely* across companies. Until we have support for a set of indexers that aligns closely with a company's tech stack, we cannot pursue those companies for adoption. There will always be a large pool of untapped users we can reach by adding support for additional languages.
 
 **Milestones:**
 
-1. Deliver precise Go code intelligence to 3 customers.
-    - ✅ [*N<sub>1</sub>*][N1]
-    - 🔄 [*N<sub>2</sub>*][N2]
-    - [*N<sub>3</sub>*][N3]
-1. Deliver precise C++ code intelligence to 3 customers.
-    - [*N<sub>4</sub>*][N4]
-    - [*N<sub>5</sub>*][N5]
-    - [*N<sub>6</sub>*][N6]
+1. 🔄 Deliver precise Go code intelligence to 3 organizations.
+    - ✅ Deliver to [*N<sub>1</sub>*][N1]
+    - 🔄 Deliver to [*N<sub>2</sub>*][N2]
+    - Deliver to [*N<sub>3</sub>*][N3]
+1. 🔄 Deliver precise C++ code intelligence to 3 organizations.
+    - Deliver to [*N<sub>4</sub>*][N4]
+    - Deliver to [*N<sub>5</sub>*][N5]
+    - Deliver to [*N<sub>6</sub>*][N6]
 1. 🔄 Support automatically indexing source code within Sourcegraph.
+    - 🔄 Provide best effort auto-indexing for supported languages [RFC 201](https://docs.google.com/document/d/1NPQs1s814LZjNXjPuavqC1N7hZR192DNtmSBmAeH9UY/edit)
```

Comment:
I've just marked this as implemented. I could use some help breaking out additional tasks as it's currently in the _long tail_ of making sure the service is reliable and useful.

Reply:
Great! I'll change it to Done and add a TBD next step